### DISCUSSION TITLE
RecorderParams by vmille25

### DISCUSSION THREAD

**Discussion by ghost** [baSSiLL/SharpAvi#54] [Q&A] [answered]
Hello, Mr. Maslov.

I am trying to implement the RecorderParams class by [vmille25](https://github.com/vmille25/CSharp-screen-recorder) using your NuGet package.  However, it was written using v2.2.1 and I am not experienced enough to be able to update it for v3.0.0.  Would you be able to make some suggestions on how to update RecorderParams properly?

Thank you for your time...

**Comment by baSSiLL** [comment, marked as answer]
Here is the diff to make it compile (and work). Basically, it comes down to

- Replace `KnownFourCCs.Codecs` references with `CodecIds`.
 - Replace the names of extension methods for creating encoding streams as they were changed in v3.

![image](https://user-images.githubusercontent.com/1926484/183032191-2b3a2a27-931c-4749-9fa4-f18caea6f0ef.png)

**Reply by ghost** [reply]
Sweet!  Thank you.
I will try.
